tests/pythonMeep.bash

```diff
@@ -0,0 +1,8 @@
+#/bin/bash
+
+#
+# test for successful loading of python meep module
+#
+cd ../python
+ln -f -s .libs/_meep.so
+python -c 'import meep'
```

Comment:
Ultimately, each of the Python tests should be a `.py` file, not a `.sh` file...